monkey/agent_plugins/exploiters/rdp/src/rdp_client.py

```diff
@@ -294,6 +307,25 @@ async def _execute_command(self, command: str):
         await self._remote_paste()
         await self._send_vk_keypress(VK_RETURN)
 
+    async def _execute_command_in_shell(self, command: str):
+        logger.debug("Executing command in shell...")
+
+        await self._execute_command("cmd.exe")
+        await asyncio.sleep(1)
+
+        await self._connection.set_current_clipboard_text(command)
+        await self._synchronize_clipboard()
+
+        alt = self._kb_layout.vk_to_scancode("VK_LMENU")
```

Comment:
It might be nice to add a comment explaining what is going on here

Reply:
+1